app/design/frontend/Inchoo/custom/Inchoo_Sample03/layout/inchoo_sample03_index_index.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="1column" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
```

Comment:
jesi sam dodavao `layout="1column"` ili si kopirao od nekuda?

Reply:
Ako se dobro sjećam kopirao sam od negdje :sweat_smile:

Reply:
ne valja slijepo kopirati, treba uvijek provjeriti što se kopira i istražiti ako je nešto nepoznanica.

Reply:
makni ga ako ti ne treba